django_codemod/visitors/core.py

```diff
@@ -0,0 +1,11 @@
+from django_codemod.constants import DJANGO_20
+from django_codemod.visitors.base import BaseSimpleRenameTransformer
+
+
+class URLResolversTransformer(BaseSimpleRenameTransformer):
+    """Resolve deprecation of ``django.conf.urls.url``."""
+
+    deprecated_in = DJANGO_20
```

Comment:
this needs to be fixed.  I pushed as-is in case any of these mistakes helped cause https://github.com/browniebroke/django-codemod/issues/124